package.json

```
@@ -15,12 +15,12 @@
   ],
   "scripts": {
     "build": "truffle compile && truffle-flattener contracts/token/Token.sol > build/TokenFlat.sol && truffle-flattener contracts/registry/IdentityRegistry.sol > build/IdentityRegistryFlat.sol && truffle-flattener contracts/registry/ClaimTopicsRegistry.sol > build/ClaimTopicsRegistryFlat.sol && truffle-flattener contracts/registry/TrustedIssuersRegistry.sol > build/TrustedIssuersRegistryFlat.sol",
-    "coverage": "solidity-coverage",
-    "test": "solidity-coverage",
+    "coverage": "truffle run coverage",
+    "test": "(ganache-cli 2>&1 > /dev/null &) && truffle test && kill $(ps -ax | grep \"ganache-cli\" | grep -v grep | cut -d' ' -f2)",
```

Why do you launch `ganache-cli` manually. `truffle test` do it automatically :thinking: 
```suggestion
    "test": "truffle test",
```